.travis.yml

```diff
@@ -13,4 +13,9 @@ jobs:
   include:
     - stage: tests
       script:
-      - make travis-runtests
+      - |
+        if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+          make travis-runtests
+        else
+          make travis-publish
```

Comment:
Also, please make sure that nightly publish only happens from the `master` branch! :smile: